examples/react-load/react/README.md

```diff
@@ -74,4 +74,4 @@ Apache 2.0 - See [LICENSE][license-url] for more information.
 
 [license-url]: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/LICENSE
 [otel]: https://opentelemetry.io/
-[otel-node]: https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node
+[otel-node]: https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-sdk-trace-node
```

Comment:
I read it again, and this README is for the react-load instrumentation.
Maybe it makes more sense to have a link to `sdk-trace-web` instead of `sdk-trace-node`?

Reply:
This is optional in case in you motivation to fix that.